tests/neg-custom-args/kind-projector.check

```diff
@@ -5,8 +5,8 @@
 -- Error: tests/neg-custom-args/kind-projector.scala:5:23 --------------------------------------------------------------
 5 |class Bar1 extends Foo[Either[*, *]] // error
   |                       ^^^^^^^^^^^^
-  |                       Type argument Either has not the same kind as its bound  <: [_$1] => Any
+  |                       Type argument Either has not the same kind as its bound [_$1]
```

Comment:
This looks wrong to me,  `[_$1]` is not a bound, it's a type parameter list.

Reply:
It's the way we print bounds now. An abstract type
```scala
type A >: Nothing <: [X] => X
```
prints as 
```scala
type A[X] <: X
```
Everything after the `A` is the bound. I think it's ultimately better this way since it is closer to the source but it might need some fine tuning.

Reply:
I see, I think if the error message did not elide the Any and printed `...its bound [_$1] <: Any` it'd be clearer